cmp/tab/TabContainerModel.js

```diff
@@ -135,16 +138,22 @@ export class TabContainerModel {
 
         if (router.isActive(route)) {
             const tab = tabs.find(t => router.isActive(route + '.' + t.id));
-
             if (tab && !tab.isActive && !tab.disabled) {
                 this.setActiveTabId(tab.id);
             }
         }
     }
 
+    markActiveTabWithRouter(id) {
```

Comment:
I think if this method takes the id we shouldn't have `ActiveTab` in the name - either `markActiveTabWithRouter()` (using this.activeTabId internally) or `markTabWithRouter(id)`.

In general, I find this name a bit confusing - maybe `forwardRouterToTab(id)` or something?

Reply:
Thanks for the careful review tom -- agree your name is clearer.   Will change and merge!